lib-src/rpc2/include/rpc2/pack_helper.h

```diff
@@ -0,0 +1,85 @@
+/* BLURB lgpl
+
+                           Coda File System
+                              Release 5
+
+          Copyright (c) 1987-2016 Carnegie Mellon University
+                  Additional copyrights listed below
+
+This  code  is  distributed "AS IS" without warranty of any kind under
+the  terms of the  GNU  Library General Public Licence  Version 2,  as
+shown in the file LICENSE. The technical and financial contributors to
+Coda are listed in the file CREDITS.
+
+
+#*/
+
+
+#ifndef _PACK_HELPER_
+#define _PACK_HELPER_
+#include <rpc2/rpc2.h>
+#include <stdlib.h>
+#include <string.h>
+#include <assert.h>
```

Comment:
stdlib.h/string.h/assert.h are not used within this header, these three can probably be removed (or should be added to the files that include pack-helper.h and need them.